### DISCUSSION TITLE
更新日志 - 22.03.4 - 20230721

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#895] [Announcements]
# 设备
* 新增支持恒领科技H88K（RK3588）
* T68M修复网口

**Comment by xixiluhui520** [comment]
放包了，我等下直接刷efi版固件

**Reply by jjm2473** [reply]
x86没更新

**Reply by xixiluhui520** [reply]
我是X64 .你是说X86-X64都没有更新是吧。系统-备份与升级-刷写新的固件，我现在能直接刷EFI版系统吗老大，保留配置能够跟通用版固件兼容吗？
在线等！

**Reply by jjm2473** [reply]
不兼容，不保留配置升级也许可以成功

**Reply by xixiluhui520** [reply]
那天忘记了磁盘格式也不对，硬盘格式要先改GPT才能刷，我刷EFI版系统了，跟通用版固件一模一样情况，不能直接刷0714，要刷0609才能连接设备，0714用命令ip addr查看接口IP都没有分配192.168.100.1的，0609可以，要0609升级0714再恢复出厂设置才能够正常使用。望老大下一版能帮我们X86_X64平台解决一下这个BUG！

**Reply by jjm2473** [reply]
不好意思，其他人都没问题，要是有问题早就有很多人反馈了。你的刷机方式可能也有问题，不可能要先改分区格式，efi固件里面包括分区表的。

**Reply by jjm2473** [reply]
你刷0714以后，恢复出厂设置看看，肯定跟0609无关。

**Reply by xixiluhui520** [reply]
老大我一个朋友跟我一样的硬件平台，固件也是跟我长期保持一致，他跟我是一模一样情况的！要想用新内核固件只能恢复出厂设置以后才能使用，刷0609恢复配置-升级-恢复出厂设置（此时LAN口无法连接也就是无内网IP，只能通过云端域名进入管理界面）-再恢复配置备份，我现在就是用的最新内核0714。我要表达的意思是新内核版直接刷不能直接食用要恢复出厂设置，你不借助0609又进不了管理界面，就是这个意思。

**Reply by jjm2473** [reply]
不可能需要借助0609，你直接刷0714，然后恢复出厂设置应该效果一样。

**Reply by xixiluhui520** [reply]
老大你一直都没有明白我的意思，直接刷0714我进不了istoreOS的管理界面呀，wan和LAN都不能连接，要刷0609接口才有输出

**Reply by special0428** [reply]
我某一个版本也跟你一样,我的解决办法就是不刷那个版本了,换个版本就好了,什么原因我也不知道.

**Reply by xixiluhui520** [reply]
老大有可能是驱动问题吗？我备份的配置文件里面有没有驱动

**Reply by xixiluhui520** [reply]
我有一个绿联的USB千兆网卡

**Reply by jjm2473** [reply]
估计是第一次启动时，某些网卡初始化较慢，系统都已经跑完初始化的脚本了，网卡还没注册，也就不会生效了。恢复出厂设置不一定要打开网页后台啊，x86不都接着键盘和显示器吗，直接执行 `firstboot -r -y` 就能恢复出厂设置了

**Reply by xixiluhui520** [reply]
知道了老大

**Comment by A-XIANGQAQ** [comment]
现在淘宝已经是n100/n200/n305的版本啦，老大啥时候更新内核呀😘

**Reply by jjm2473** [reply]
等到 openwrt 的 23.05 正式发布的时候才可能更新，现在还在rc阶段
